packages/lodestar/src/api/impl/errors/api.ts

```diff
@@ -5,3 +5,14 @@ export class ApiError extends Error {
     this.statusCode = statusCode;
   }
 }
+
+export class StateNotFound extends ApiError {
+  constructor() {
+    super(404, "State not found");
```

Comment:
can we put state id here so that we have this example error:
```
{
  "code": 400,
  "message": "Invalid state ID: current"
}
```

Reply:
That's not the same error, this is when there is correct state id but we don't have it in database.